batcher/aligned-batcher/src/lib.rs

```diff
@@ -833,7 +833,7 @@ async fn send_batch_inclusion_data_responses(
             let response = ResponseMessage::BatchInclusionData(batch_inclusion_data);
 
             let serialized_response =
-                serde_json::to_vec(&response).expect("Could not serialize response");
+                cbor_serialize(&response).expect("Could not serialize response");
```

Comment:
lets handle this correctly too

Reply:
Fixed [f0f5354](https://github.com/yetanotherco/aligned_layer/pull/746/commits/f0f53540a95ec50c447ed30c61ff3eefb02c3701)